### ISSUE TITLE
Support `subprocess.DEVNULL` in cptbox

### ISSUE THREAD

**Issue by Xyene** [DMOJ/judge-server#962]
It wasn't present in Python 2, hence why we have `NullStdoutMixin`. Nowadays, we could do it directly in `tracer.py`, and drop `NullStdoutMixin`.

**Comment by michelececcacci** [comment]
is this still open?

**Comment by Xyene** [comment]
Yes, any tickets that are not closed are still open. We try to keep the issue tracker up-to-date.
